packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -830,7 +830,7 @@ export default mixins(
 			},
 			valueChanged (value: string[] | string | number | boolean | Date | null) {
 				if (this.parameter.name === 'nodeCredentialType') {
-					this.prepareScopesNotice(value as string);
+					this.activeCredentialType = value as string;
```

Comment:
what happens if I open credential directly from credential selector here and delete it?

Reply:
If you have select `Existing Credential Type` → (cred type) → (actual cred), click the pencil, delete the cred and return to NDV, the cred disappears and field is flagged in red as requiring a cred, so validation works as expected. Tested with OAuth and plain auth, and tested with 1 cred existing per cred type and 2 creds existing per cred type.